Four New Accents. and One Expanded Upon

CorruptOmega opened the pull request:
<!-- IT'S NOT WIZDENS REPO, IF YOU WANT TO ADD YOUR CHANGES ON ALL SERVERS, CREATE PR TO WIZDENS REPO -->

## Short description
<!-- What do you propose to change with your PR? -->
Adding four new accents to the trait selection. 

Cost 2 Points:
Archaic (Shakespeare)
Scottish

Cost 1 Point:
Nerd

Cost 0 Point:
Polite

Polite accent is also a smitable option: (Speak Proper Language) in case someone really can't stop cussing.

Pirate accent has had more words included in their list of word replacements, because they didn't have much before and I was going to add a "Sailor" accent before deciding to just shove it into one that already existed. Win Win.

Omni Accent Smite has been given Archaic, Scottish, Pirate, and Nerd. Because it's Omni. But not Polite because that's a separate option now.

## Why we need to add this
<!-- What is the reason for adding these changes? Please post links to Discussions as well as Bug Reports here. Please describe how this will change the game balance. -->
More variety for accents? Don't know how well they will play with the actual voices in game, but we'll see. Archaic and Scottish were suggestions to be brought in/brought back before.
Nerd is just because someone wanted a dumb language to just make words unnecessarily long, like I made this sentence.
Polite is just as a light chat filter for those who want to play a more polite character. Or if someone needs to be smited to curb some of their language. :)
Expanding pirate language just to improve it.

## Media (Video/Screenshots)
<!--
If your PR contains in-game changes you must provide screenshots/videos of the changes.
-->

<img width="636" height="400" alt="Accent Update" src="https://github.com/user-attachments/assets/86bc59f5-0305-48f5-8e45-124930adf772" />


## Checks
<!-- check boxes for faster reviewing of your PR -->

- [X] I do not require assistance to complete the PR.
- [X] Before posting/requesting review of a PR, I have verified that the changes work.
- [X] I have added screenshots/videos of the changes, or this PR does not change in-game mechanics.
- [X] I affirm that my changes are licensed under the [Starlight Fork License](https://github.com/ss14Starlight/space-station-14/blob/Starlight/LICENSE-Starlight.TXT) and grant permission for use in this repository under its conditions.

**Changelog**
:cl: CorruptOmega
- add: Added Archaic (Shakespeare) accent. // To be, or not to be, that is potentially the answer.
- add: Added Scottish accent. // Whit ye mean ae chiel cuid no' speak Scottish afore?
- add: Added Nerd accent. // Cause the Grand Admiral leads the Orbital Habitation Module.
- add: Added Polite accent. // You watch your language now.
- add: Admin Smite Option. Speak Proper Language - Gives Polite accent.
- tweak: Expanded Pirate accent. It was lacking in some of it's accent.
- tweak: Omni Accent also got the new accents included (Not Polite, since I separate that out)

<!--
If you want the players to know about changes made in this PR, specify them using the template outside the comment. Short and informative.

:cl: STARLIGHT TEAM
- add: Added Starlight.
- remove: Removed SS13.
- tweak: Changed SS14.
- fix: Fixed Rinary.
-->

CorruptOmega committed:
Added 2 Accents - Archaic (Shakespear) and Scottish

CorruptOmega committed:
Adding in a basic swear replacement

CorruptOmega committed:
Slight typo fix :)

CorruptOmega commented:
Was potentially thinking of adding another accent or two, but nothing came to mind, and I can always add later.

CorruptOmega committed:
Update Pirate, Add Nerd and Polite

Also added smitable action of giving people proper language.

TheRealSlimRedwoods commented:
no fucking way you added the "erm actually" accent

CorruptOmega commented:
I also made it so when you say well, it translates into well actually, . Heheh

AutumnalModding commented:
`an asshole` should be -> `a friend`, not `an friend` IMO

CorruptOmega committed:
Update polite.ftl

CorruptOmega committed:
Update word_replacements.yml

CorruptOmega commented:
> `an asshole` should be -> `a friend`, not `an friend` IMO

It was separated words. So 'asshole' was translated to 'friend'. But I added another one on for that separate version you pointed out. :)

TheRealSlimRedwoods commented:
> I also made it so when you say well, it translates into well actually, . Heheh

"well actually-" BANG BANG BANG BANG BANG BANG BANG

CorruptOmega committed:
Update nerd.ftl

Changing out some options/cleaning up.

CorruptOmega committed:
Update nerd.ftl (again)

Silly duplicate lines.

CorruptOmega committed:
Update nerd.ftl

Separated out some stuff that got scuffed.

CorruptOmega committed:
Update word_replacements.yml

Adding two more into nerd accent

levinaHR commented:
if 'well' turns into 'well actually', 'actually' should turn into 'erm actually' and that would be perfection

CorruptOmega commented:
> if 'well' turns into 'well actually', 'actually' should turn into 'erm actually' and that would be perfection

-cries- yall are trying to harm me with changes.

CorruptOmega committed:
Update nerd.ftl (For the very fourth time) 

Another line added.

CorruptOmega committed:
Update word_replacements.yml

Adding one more line.

torturedbymemes commented:
A SMITE? oohhhhhh this'll be funny

CorruptOmega commented:
> A SMITE? oohhhhhh this'll be funny

Indeed! A smite! Also added the new accents to Omni Accent since I was there anyways.

CorruptOmega committed:
Update AdminVerbSystem.Smites.cs

Added Nerd Accent to the omni accent smite, cause I forgot to after completing it.

CorruptOmega committed:
Merge branch 'starlight-dev' into Traits-Accents

Cerzix commented:
Is it possible to get a nerd accent without the w change? sort of a snobby noble accent?

CorruptOmega commented:
> Is it possible to get a nerd accent without the w change? sort of a snobby noble accent?

Unfortunately I decided to use a felinoid character when I applied the nerd accent. Nerd Accent does not change L's/R's into W's, that's the felinoid natural one.

Cerzix commented:
> > Is it possible to get a nerd accent without the w change? sort of a snobby noble accent?
> 
> Unfortunately I decided to use a felinoid character when I applied the nerd accent. Nerd Accent does not change L's/R's into W's, that's the felinoid natural one.

I see! thats good lmao

ShadesMars commented on Resources/Textures/_Starlight/Interface/AdminActions/proper.png:
What the fuck is this?

CorruptOmega replied:
![shopping.jpg](https://github.com/user-attachments/assets/fbf5a9a4-bf7d-44a0-b82c-3141a2c0d230)

Heheheh

ShadesMars reviewed (approved):
That icon... damn
